code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -363,7 +365,7 @@
 			cyborg.cell.use(powercost)
 		else
 			cyborg.cell.use(5)
-		repair_tick = 0
+		next_repair = world.time + repair_cooldown SECONDS
```

Comment:
Using these defines look pretty bad outside of simple defines to me. Not very helpful with understanding the meaning either since the result is not in seconds.

Reply:
Alright, fair enough